src/site/checkstyle.xml

```diff
@@ -89,7 +89,8 @@
         <!-- Checks for imports                              -->
         <!-- See http://checkstyle.sf.net/config_import.html -->
         <module name="AvoidStarImport"/>
-        <module name="IllegalImport"/> <!-- defaults to sun.* packages -->
+        <module name="IllegalImport"/>
+        <!-- defaults to sun.* packages -->
```

Comment:
fixed this in d1eaa44, but it was indead the auto formatting that wants to change it´s location